analyzers/src/SonarAnalyzer.CFG/ShimLayer/IOperationWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#nullable enable
+
+namespace StyleCop.Analyzers.Lightup
+{
+    using Microsoft.CodeAnalysis;
+
+    public interface IOperationWrapper
+    {
+        IOperation? WrappedOperation { get; }
+
+        ////IOperationWrapper Parent { get; }
```

Comment:
In general I saw a lot of commented code. Should we remove it?

Reply:
I think we should keep the manual changes on this code as few as possible. Since we don't actually maintain the it the commented code should not be a problem.